src/main/resources/com/google/jenkins/plugins/computeengine/InstanceConfiguration/help-networkInterfaceIpStackMode.html

```diff
@@ -0,0 +1,18 @@
+<!--
+ Copyright 2020 Google LLC
```

Comment:
are you sure of this? :) 
this should be the company you're working for (if changes made for $DAYJOB) or under your name

Reply:
fixed in 3fd1798